src/pages/LoginPage.js

```diff
@@ -1,23 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState} from 'react';
 import image from '../images/image-1.png';
 import { useNavigate} from 'react-router-dom';
 
 
 function Login() {
     const [Username, setUsername] = useState('');
     const [Password, setPassword] = useState('');
-    const navigate = useNavigate();;
-
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
+    const navigate = useNavigate();
+   
+    
     const handleUsernameChange = (event) =>{
         setUsername(event.target.value);
        
     }
     const handlePasswordChange = (event) =>{
         setPassword(event.target.value);
     }
-    const handleLogin = (event) =>{
+     const handleLogin = (event) =>{
         if( Username === "admin" && Password === 'admin'){
-            navigate('/tasks');
+            localStorage.setItem('isLoggedIn', 'true');
```

Comment:
Let's set `username` as `admin`:
```
localStorage.setItem('username', 'admin');
```
Because `isLoggedIn` is very generic and we want to save an information for a specific user.